test/tcp/net_test.go

```diff
@@ -0,0 +1,19 @@
+package functions
```

Comment:
We plan to use the test directory to store e2e tests and performance tests later. net_test.go and serviceProxy_test.go seem to be unit tests. It is recommended to keep the same location as the source code. You can refer [example](https://github.com/kubeedge/kubeedge/tree/5fce8601e71fa3b71c9d1c2846bb118fce611b74/cloud/pkg/dynamiccontroller/messagelayer).

Reply:
so sorry, i have deleted net_test.go, and move serviceproxy_test.go to the location of the serviceproxy.go, and keep the table-driven style.